Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 
 let package = Package(
     name: "VonageClientSDKVideo",
-    platforms: [.iOS(.v13)],
+    platforms: [.iOS(.v14)],
```

Comment:
pods is v13.  We should update that too and update dev center release notes too.

Reply:
Reason for  bump?